src/jira/client/jira-client.ts

```diff
@@ -421,6 +452,23 @@ const updateIssueKeysFor = (resources, func) => {
 	return resources;
 };
 
+/**
+ * Runs the mutatingFunc on the association values field for each entity resource
+ * Assumption is that the transformed resource only has one association which is for
+ * "issueIdOrKeys" association.
+ */
+const updateIssueKeyAssociationValuesFor = (resources, mutatingFunc) => {
+	resources.forEach((resource) => {
+		resource.associations[0].values = mutatingFunc(resource.associations[0].values)
```

Comment:
We should probably check that associations and non-null and has a length of not 0 or else the whole function will blow up